cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -82,10 +82,14 @@ func TestImagesListRunWithCustomConfigPath(t *testing.T) {
 				constants.CurrentKubernetesVersion.String(),
 			},
 			configContents: []byte(dedent.Dedent(fmt.Sprintf(`
-				apiVersion: %s
-				kind: ClusterConfiguration
-				kubernetesVersion: %s
-			`, kubeadmapiv1.SchemeGroupVersion.String(), constants.CurrentKubernetesVersion))),
+apiVersion: %s
+kind: InitConfiguration
+nodeRegistration:
+  criSocket: "unix:///var/run/containerd/containerd.sock"
```

Comment:
can we use https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/constants/constants.go#L269 instead of valid sockets everywhere?

unit tests should pass even if no known container runtime socket is present on the host.

Reply:
all addressed now.